conf/gulp-tasks/bundle/bundle.js

```diff
@@ -29,16 +29,22 @@ exports.modernBundle = function (callback, outputConfig, bundleName, locale, lib
     input: './src/answers-umd.js',
     output: outputConfig,
     plugins: [
-      resolve(),
+      resolve({ browser: true }),
       commonjs({
-        include: [/node_modules/, /answers-core/]
+        include: [/node_modules/]
       }),
       babel({
         babelrc: false,
-        exclude: [/node_modules/, /answers-core/],
+        exclude: [/node_modules/],
         presets: ['@babel/env']
       })
-    ]
+    ],
+    onwarn: function(warning) {
```

Comment:
I think I'd rather use the commonjs bundle than have this onwarn, assuming the bundle sizes are basically the same

Reply:
ya, they're nearly the same. I'll update.

Leaving it as it is now would suppress the warning for other dependencies, and we probably don't want that